serverless/serverless-tracing.adoc

```diff
@@ -0,0 +1,31 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-tracing"]
+= Tracing requests
+:context: serverless-tracing
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Using Jaeger with {ServerlessProductName} allows you to enable _distributed tracing_ for your serverless applications on {product-title}.
+
+Distributed tracing records the path of a request through the various services that make up an application.
+
+It is used to tie information about different units of work together, to understand a whole chain of events in a distributed transaction.
+The units of work may be executed in different processes or hosts.
+
+Developers can visualize call flows in large architectures with distributed tracing. which is useful for understanding serialization, parallelism, and sources of latency.
+
+For more information about Jaeger, see the documentation on xref:../jaeger/jaeger_arch/rhbjaeger-architecture.adoc#rhbjaeger-architecture[Jaeger architecture].
+
+== Using Jaeger with {ServerlessProductName}
+
+To use Jaeger with {ServerlessProductName}, you will need to install the *Elasticsearch Operator*, and then the *Jaeger Operator*.
```

Comment:
This is techically not true I believe. Should we link to the jaeger docs here? They state you can use jaeger without elasticsearch but you should install elasticsearch if you want to use it in production.

See https://docs.openshift.com/container-platform/4.4/jaeger/jaeger_install/rhbjaeger-installation.html